manifests/init.pp

```diff
@@ -81,9 +81,6 @@
   Wildfly::Config_file                               $config                       = 'standalone.xml',
   Wildfly::Config_file                               $domain_config                = 'domain.xml',
   Wildfly::Config_file                               $host_config                  = 'host.xml',
-  String                                             $java_xmx                     = '512m',
```

Comment:
I'm not sure if this is a good idea. When we make those parameters optional, and people didn't set them in hiera, the xms/xmx options will vanish from their standalone.conf and potentially break their service. What's the issue with those parameters + `$java_opts`?

Reply:
A customer needs to not set `java_maxmetaspace_size` but another Java option. If we set the `java_maxmetaspace_size` to an empty string, there will still be Config artefacts in the config file. (`-XX:MaxMetspaceSice=`).
This patch should allow users to either use the `java_xmx`, `java_xms` and `java_maxmetaspace_size` parameter or to add all Java options to `java_opts` Array or String.

Reply:
Fixed